parsec/core/gui/central_widget.py

```diff
@@ -105,6 +112,28 @@ def handle_event(self, event, **kwargs):
             self.new_notification.emit("WARNING", _("NOTIF_WARN_INCOMPATIBLE_VERSION"))
         elif event == "mountpoint.stopped":
             self.new_notification.emit("WARNING", _("NOTIF_WARN_MOUNTPOINT_UNMOUNTED"))
+        elif event == "mountpoint.remote_error":
+            # TODO: find a way to provide the involved path
+            # path = kwargs["path"]
```

Comment:
I guess the path should be provided by the mountpoint:
```python
        with translate_error(self.event_bus, path):
            stat = self.fs_access.entry_info(path)
```
Even better, the context could be automatically passed for all operation callbacks so that the name of the operation is also accessible:
```python
    def __call__(self, name, path, *args, **kwargs):
        with translate_error(self.event_bus, name, path):
            return super().__call__(name, FsPath(path), *args, **kwargs)
```
Now that I think about it, I don't see the point of using the `translate_error` context for each specific fs call since it is meant to re-raise an exception anyway.

Reply:
ec3e54b